docs/release-notes/NuGet-6.14.md

```diff
@@ -2,27 +2,130 @@
+
+* NuGet 6.13.2.1 does no longer support prerelease versions - [#14219](https://github.com/NuGet/Home/issues/14219)
+
+* Badly specified framework leads to an uninformative error - [#14216](https://github.com/NuGet/Home/issues/14216)
+
+* Restore should fail more quickly when using http sources - [#14210](https://github.com/NuGet/Home/issues/14210)
+
+* README spins indefinitely if the Readme URI does not result in a readme - [#14201](https://github.com/NuGet/Home/issues/14201)
+
+* Error in Visual Studio if Path contains directory you do not have permission to view - [#14192](https://github.com/NuGet/Home/issues/14192)
+
+* list package doesn&#39;t with a solution argument in 9.0.201 - [#14177](https://github.com/NuGet/Home/issues/14177)
+
+* Focus shifts from Output window to Error List after Clear NuGet Locals command - [#14157](https://github.com/NuGet/Home/issues/14157)
+
+* NuGet adding a bunch of generally unuseful information to VS activity log - [#14153](https://github.com/NuGet/Home/issues/14153)
+
+* NuGet authentication plug-in discovery fails when environment variable has trailing semicolon - [#14144](https://github.com/NuGet/Home/issues/14144)
+
+* nuget.exe fails to find Microsoft.VisualStudio.SolutionPersistence.dll - [#14136](https://github.com/NuGet/Home/issues/14136)
+
+* NuGet restore writes dgspec too frequently - [#14135](https://github.com/NuGet/Home/issues/14135)
+
+* dotnet list package does not display resolved versions when AuditSources are used - [#14116](https://github.com/NuGet/Home/issues/14116)
+
+* The REAMD tab always shows “Loading README” for the latest version of the package in the detail panel of PM UI - [#14098](https://github.com/NuGet/Home/issues/14098)
+
+* [Bug Bash] The second time clicking ‘Installed’ tab for a remote source which doesn’t allow for downloading a README shows the README tab - [#14097](https://github.com/NuGet/Home/issues/14097)
+
+* HttpFileSystemBasedFindPackageByIdResource.ConsumeFlatContainerIndexAsync allocates significantly more than necessary - [#14095](https://github.com/NuGet/Home/issues/14095)
+
+* UnresolvedMessages.GetMessageAsync is allocating more heavily than necessary - [#14094](https://github.com/NuGet/Home/issues/14094)
+
+* New dependency resolver does not properly detect a cycle with a transitive dependency with the same name as the root project - [#14052](https://github.com/NuGet/Home/issues/14052)
+
+* dotnet nuget why does not give an error if only a project path was specified - [#14030](https://github.com/NuGet/Home/issues/14030)
+
+* Cleanup ServiceProviderExtensions, remove GetFreeThreadedServiceAsync - [#14007](https://github.com/NuGet/Home/issues/14007)
+
+* Value cannot be null. Parameter name: versionRange when opening the PM UI - [#13933](https://github.com/NuGet/Home/issues/13933)
+
+* Reenable new algorithm resolution with lock files - [#13800](https://github.com/NuGet/Home/issues/13800)
```

Comment:
We should clarify that the new algorithm only supports lock files when using the .NET 10 SDK, which is in previews right now.